File: URI-cpp/1074.cpp
#include <iostream>
#include <iomanip>
#include <math.h>
#include <string>

using namespace std;

int main()
{
    int n,val;
    string name;
    cin >> n;
    for (int i = 0; i < n; i++)
    {
        cin >> val;
        if (val==0)
        {
            name += "NULL";
        }
        else if (val%2==0)
        {
            name += "EVEN ";
        }
        else if (val%2!=0)
        {
            name += "ODD ";
        }
        if (val>0)
        {
            name += "POSITIVE";
        }
        else if (val<0)
        {
            name += "NEGATIVE";
        }
        cout << name << endl;
        name = "";
    }

    return 0;
}

File: URI-py/1074.py
# -*- coding: utf-8 -*-
n = int(input())
for _ in range (0,n+1):
    a = int(input())
    if a == 0:
        print("NULL")
    elif a % 2 == 0:
        if _ > 0:
            print("EVEN POSITIVE")
        else:
            print("EVEN NEGATIVE")
    elif a % 2 != 0:
        if _ > 0:
            print("ODD POSITIVE")
        else:
            print("ODD NEGATIVE")
